liferay-lima/liferay-portal
# LPS-150268 Index Scheduled Blogs

Motivation
=======
Scheduled Blog Entries doesn't appear in the filter when Search is used:

[LPS-150268](https://issues.liferay.com/browse/LPS-150268)

The current behaviour is:

1. Publish a Blog with a Display Day in the future
1. Use the search bar to search for it
1. The blog does NOT appear in the results

https://user-images.githubusercontent.com/5776822/187410438-c0c65c43-5f1c-4041-a618-ee9ba30c4070.mp4


Proposed Solution
========
Index scheduled blogs too in ES

Steps to Verify:
----------------
1. Publish a Blog with a Display Day in the future
1. Use the search bar to search for it
1. The blog appears in the results

Screencaps:
---------------

https://user-images.githubusercontent.com/5776822/187411030-4a44c992-ed6b-47c3-9ccd-16ce793f244f.mp4

Notes:
---------------

No integration tests have been included because I have not been able to understand correctly how your search tests works, but if you give me some direction I can add them.
